components/textbox/src/_mixin.scss

```
@@ -276,6 +276,16 @@ $utrecht-support-prince-xml: false !default;
   --utrecht-textbox-value-max-length: 7;
 }
 
+@mixin utrecht-textbox--postbus-nummer-size {
+  /* A postbusnummer consists of four digits. */
+  --utrecht-textbox-value-max-length: 4;
+}
+
+@mixin utrecht-textbox--postcode-size {
```

Vanwege `autocomplete="postal-code"`

```suggestion
@mixin utrecht-textbox--postal-code-size {
```